src/main/java/com/moodybluez/enterprise/service/IMoodService.java

```diff
@@ -1,12 +0,0 @@
-package com.moodybluez.enterprise.service;
-
-import com.moodybluez.enterprise.dto.Mood;
-
-public interface IMoodService {
-    /**
-     * fetch a mood with a given ID
-     * @param id unique identifier for a mood
-     * @return the matching mood, or null if no matches found
-     */
-    Mood fetchById(int id);
-}
```

Comment:
Removed these files, because the logic here should really be in the DAO layer, which I have created and that includes necessary methods created in these service layer files.